app/src/main/res/values/arrays.xml

```diff
@@ -11,32 +11,26 @@
         <item>system_default</item>
     </string-array>
     <string-array name="language_settings">
-        <item>@string/SETTINGS_LANGUAGE_SYSTEM_DEFAULT</item>
```

Comment:
What happens if a user has `System default` as their setting and enter the settings screen? 🤔

Reply:
Good catch! It says `Inget angett` and nothing is selected in the dialog

Reply:
Perhaps we could, in `HedvigApplication::onCreate`, migrate these members to the 'default setting' that you added in `Language.kt`? 😉